lib/diagnostic.js

```diff
@@ -45,11 +45,15 @@ let scopes = [/* your response here */];
 // context of `alderaan`. In other words, we want to *reference* the `fire`
 // method and temporarily bind it (or, change `this`) to `alderaan` before
 // executing it.
-//
+
+console.log(alderaan);
 let fireOnAlderaan = function () {
-  // your code here
+  let boundMethod = deathStar.fire.bind(alderaan);
+  boundMethod();
```

Comment:
that's one way to do it. you can also use `.call` instead to call the bound function immediately